.evergreen/serverless/create-instance.sh

```diff
@@ -105,12 +106,12 @@ if [ $SERVERLESS_URI = "null" ]; then
   exit 1
 fi
 
-SERVERLESS_MONGODB_VERSION=$(curl -sS \
+RESP=$(curl -sS \
   --digest -u "${ATLAS_PUBLIC_API_KEY}:${ATLAS_PRIVATE_API_KEY}" \
   -X GET \
-  "${ATLAS_BASE_URL}/groups/${ATLAS_GROUP_ID}/serverless/${DEPLOYMENT_NAME}" \
-  | jq -r '.mongoDBVersion'
+  "${ATLAS_BASE_URL}/groups/${ATLAS_GROUP_ID}/serverless/${DEPLOYMENT_NAME}"
 );
+SERVERLESS_MONGODB_VERSION=$($PYTHON -c "import json;d=json.loads('${RESP}');print(d['mongoDBVersion'])")
```

Comment:
I think we need to use `find_python3` in this function to make sure this is defined, no?

Reply:
The find-python3.sh script is invoked in L28 in this file, is there anything else that needs to be done? I'll add that this ran correctly after the change where before it aborted with the `jq: command not found` error

Reply:
That invocation only imports the function, I think we still need this block to be self-contained:

```bash
  echo "Finding Python3 binary..." 1>&2
  PYTHON="$(find_python3 2>/dev/null)"
  echo "Finding Python3 binary... done." 1>&2
```

Reply:
Ah, I see. I missed that from the other file, thanks for pointing that out!